module-2/backend/week_one.md

```diff
@@ -5,30 +5,58 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+  Get, Post, Redirect, Render, Put
 2. What is Sinatra?
+  A local hosting service to test http functionality.
 4. What is MVC?
+  Models, views, controllers
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+  So that other coders/people can look in to our app and make sense of it.
 6. What types of variables are accessible in our view templates without explicitly passing them?
+  instance variables, in that specific block
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
+    ** @name = 'Mr. Ed'
+    ** @count = 1 **
     erb :index
   end
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed`?
+  see above
 9. What's the purpose of ERB?
+  to provide an html file that can process ruby code in interpolation <% %>
 10. Why do I need a development AND test database?
+  So you can specify what you want to test and not have to test through all of the data in the dev database
 11. What's responsive design?
+  A front end design that responds quickly to requests
 12. What is CRUD and why is it important?
+  Create, Review, Update, Delete : provides a structure for basic web database interaction
 13. What does HTTP stand for? 
+  Hyper text transfer protocol
 14. What are the two ways to interpolate Ruby in an ERB view template? What's the difference between these two ways?
+  <%= and <% brackets. The bracket with the = sign included will output the results to the html page
 15. What's an ORM?
+  Object Relational Mapper
 16. What's the most commonly used ORM?
+  not sure
 17. Let's say we have an application with restaurants. There are seven verb + path combinations necessary to provide full CRUD functionality for our restaurant application. List each of the seven combinations, and explain what each is for.
-18. What's a migration? 
+  See all - Get/Render

```

Comment:
what you have here is correct... it's just missing the name `restaurants`, so the following is more accurate:

get '/restaurants' -> see all restaurants
get '/restaurants/:id' -> see one restaurant
get '/restaurants/new' -> see form to create new restaurant
post '/restaurants' -> create a new restaurant in the db
get '/restaurants/:id/edit' -> see form to edit an existing restaurant
put '/restaurants/:id' -> update restaurant in db
delete '/restaurants/:id' -> delete restaurant from db